pkgs/by-name/ho/hosts-bl/package.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "Hosts-BL";
```

```suggestion
  pname = "hosts-bl";
```
I think these are supposed to match the package name and not have uppercase letters.